app/partials/errorDetail-Q029.html

```diff
@@ -1,43 +1,41 @@
-                <th scope="col" class="county_code">Reported County Code</th>
-                <th scope="col" class="census_tract">Reported Census Tract</th>
-                <th scope="col" class="recommended">Recommended MSA/MD</th>
-                <td><select ng-model="selectAll"><option>Select...</option><option>All of the listed MSA/MD's should be updated</option><option>None of the listed MSA/MD's should be updated</option></select></td>
-            </tr>
-        </thead>
-        <tbody>
-            <tr ng-repeat="item in error.errors | paginate:start():end()">
-                <th scope="row" class="id"><label for="lar-{{item.properties['LAR number']}}">{{item.properties['LAR number']}}</label></th>
-                <td class="state_code">{{item.properties['Reported State Code']}}</td>
-                <td class="county_code">{{item.properties['Reported County Code']}}</td>
-                <td class="census_tract">{{item.properties['Reported Census Tract']}}</td>
-                <td class="recommended">{{item.properties['Recommended MSA/MD']}}</td>
-                <td><select id="lar-{{item.loanNumber}}"><option>Select...</option><option>Yes, the listed MSA/MD should be updated</option><option>No, the listed MSA/MD should not be updated</option></select></td>
-            </tr>
-        </tbody>
-    </table>
+<table id="Q029" class="error-detail" ng-if="editError">
+    <thead>
+        <tr>
+            <th scope="col" class="id">LAR Number</th>
+            <th scope="col" class="state_code">Reported State Code</th>
+            <th scope="col" class="county_code">Reported County Code</th>
+            <th scope="col" class="census_tract">Reported Census Tract</th>
+            <th scope="col" class="recommended">Recommended MSA/MD</th>
+            <td><select ng-model="selectedAnswer" ng-change="selectAll(selectedAnswer)"><option value="0">Select...</option><option value="1">All of the listed MSA/MD's should be updated</option><option value="2">None of the listed MSA/MD's should be updated</option></select></td>
+        </tr>
+    </thead>
+    <tbody>
+        <tr ng-repeat="item in editError.errors | paginate:start():end()">
+            <th scope="row" class="id"><label for="lar-{{item.properties['LAR number']}}">{{item.properties['LAR number']}}</label></th>
+            <td class="state_code">{{item.properties['Reported State Code']}}</td>
+            <td class="county_code">{{item.properties['Reported County Code']}}</td>
+            <td class="census_tract">{{item.properties['Reported Census Tract']}}</td>
+            <td class="recommended">{{item.properties['Recommended MSA/MD']}}</td>
+            <td><select id="lar-{{$index}}" ng-model="selects[start()+$index]"><option value="0">Select...</option><option value="1">Yes, the listed MSA/MD should be updated</option><option value="2">No, the listed MSA/MD should not be updated</option></select></td>
```

Comment:
Oops! This select should actually be labeled by the Recommended MSA/MD value. Can you please move the label from the LAR number to the MSA/MD?